src/vm/wren_core.wren

```diff
@@ -323,6 +323,41 @@ class List is Sequence {
     return other
   }
 
+  sort() { sort {|a, b| a < b } }
+
+  sort(comparer) {
+    if (!(comparer is Fn)) {
+      Fiber.abort("Comparer must be a function.")
+    }
+    quicksort_(0, count - 1, comparer)
+    return this
+  }
+
+  quicksort_(a, b, comparer) {
+    if (a < b) {
```

Comment:
@mathewmariani this is using `<` despite comparer being provided. 
I would expect `if(comparer.call(a, b)) {` to be the condition instead?

Reply:
Actually `a` and `b` are the start and end indexes of the range that's being sorted. Since the range is inclusive, the comparison is checking whether the range has more than 1 element.

Maybe there could be more intuitive names for these variables.

Reply:
Oh right, I misread there. I agree with better naming.

Reply:
I was thinking `i` and `j` since that's what most algorithms & data-structure books might use. But, `start` and `end` can be used as well.